core/settings.py

```diff
@@ -62,6 +72,18 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
 
+if DEBUG_TOOLBAR_ENABLED:
```

Comment:
This is well implemented. 
To make it even more robust I suggest placing "debug_toolbar" and the debug_toolbar middleware string in a constant and using it to increase readability & maintainability `DEBUG_TOOLBAR = "debug_toolbar"`